index.html

```diff
@@ -1165,25 +1209,58 @@ <h3>Issuer</h3>
 machine-readable information about the <a>issuer</a> that can be used to
 <a>verify</a> the information expressed in the <a>credential</a>.
           </dd>
+        </dl>
+
+        <pre class="example nohighlight" title="Usage of issuer property">
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://example.com/examples/v1"
+  ],
+  "id": "http://example.edu/credentials/3732",
+  "type": ["VerifiableCredential", "UniversityDegreeCredential"],
+  <span class="highlight">"issuer": "https://example.edu/issuers/14"</span>,
+  "issuanceDate": "2010-01-01T19:23:24Z",
+  "credentialSubject": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "degree": {
+      "type": "BachelorDegree",
+      "name": "Bachelor of Science in Mechanical Engineering"
+    }
+  },
+  "proof": { ... }
+}
+        </pre>
+      </section>
+
+      <section>
+        <h3>Issuance Date</h3>
+
+        <p>
+The date and time when the <a>credential</a> was issued MUST be
+expressed using the following <a>property</a>:
+        </p>
+
+        <dl>
           <dt><var>issuanceDate</var></dt>
           <dd>
 The value of the <code>issuanceDate</code> <a>property</a> MUST be a string
-value of an [[!ISO8601]] combined date and time string representing the date
+value of an [[!RFC3339]] combined date and time string representing the date
```

Comment:
The reviewer recommended we specify UTC in addition to RFC3339. could we add that as a SHOULD?